### PR TITLE
add default documentation formats

### PR DESCRIPTION
**Pull request by jdeniau**
See https://github.com/api-platform/core/pull/1891

### CODE REVIEW

File: core/configuration.md [hunk 1]
```diff
@@ -179,6 +179,8 @@ api_platform:
         jsonld:
             mime_types: ['application/ld+json']
 
+    # The list of enabled documentation formats. The first one will be the default.
+    documentation_formats:
         json:
```
Comment (dunglas) [inline core/configuration.md]: jsonld should be there too right?

Comment (jdeniau) [reply core/configuration.md]: Hmmm, if you think it is useful to output jsonld, I can add it back, but as swagger understand json, I removed it

Comment (dunglas) [reply core/configuration.md]: If we disable the Hydra documentation (`jsonld` format), the admin and the PWA generator will not work anymore.

**Comment by jdeniau** [comment]
OK I will add it back

Le mer. 25 avr. 2018 à 16:48, Kévin Dunglas <notifications@github.com> a
écrit :

**Pull request closed without merging**